### ISSUE TITLE
paper and code contradicting Results 

### ISSUE THREAD

**Issue by saikalyan9981** [sinhasam/vaal#7]
I tried to replicate your results in your paper for cifar 10 with the help of your code.I found the following things in paper and code contradicting , can you help me out please?
Issues:
1. In paper you mentioned for cifar 10, you trained with 100 epochs. But code was with respect to iterations.
I resolved it as for  budget b, 
no. of iterations = (b*( no. of epochs))/(batch size) 
So number of iterations grows as split size increases.

2. If I remove the Vaal component, and perform random Sampling, I am not getting the results of your paper.
I am getting  (63,70,76,79,79,81) accuracies for each split which are on par when I ran using Vaal. and quite high compared to your paper. 
Here is the link of public Kaggle kernel of implementation of random sampling:
https://www.kaggle.com/bestfitters/random-sample-learning

**Comment by sinhasam** [comment]
1. 100 epochs often lead to some variance in the performance, so the code was released with a set number of iterations, which led to smaller variance. 

2. I will recreate the results of the paper and re-check the training hyperparameters, used for the baselines and for VAAL, and update the code, if needed. Thank you for bringing it up to my attention.

**Comment by IgorSusmelj** [comment]
Hi, I really like the paper and also tried to reproduce the results. To me it looks like the VAE is not learning any meaningful reconstructions. I changed the code to look at reconstructed images and they are just random noise. Also the accuracy results seem not to be better than random. 
Any idea what could be the problem?

**Comment by sinhasam** [comment]
Yes, in my experience, the VAE didn’t reproduce good reconstructions or generated samples. My intuition is that the reconstruction loss adds some useful regularization for the model. 

In my experience, VAAL is very sensitive to hyper-parameters so tuning the hyperparameters could be useful. Feel free to send me an email, for any specific concerns.
